src/metabase/lib/remove_replace.cljc

```diff
@@ -191,6 +193,48 @@
      (remove-join query stage-number target-clause)
      (remove-replace* query stage-number target-clause :remove nil))))
 
+(defn- expression-role
+  [an-expression]
+  (if (-> an-expression lib.options/options :lib/expression-name)
+    :custom-column
+    :custom-aggregation))
+
+(let [expression-validator (mc/validator :metabase.lib.schema.expression/expression)]
```

Comment:
`def`s not at the top level can make CLJS sad. Is there any reason not to make this a `(def ^:private expression-validator ...)` and let `tweak?` be at the top level?

Reply:
No serious reason, just that I didn't intend this validator to leak the function even. I'll change it. What problems does this cause in CLJS?